wpeverest/user-registration
# UR-1049 profile details updated email to user

### All Submissions:






### Changes proposed in this Pull Request:



Previously, we didn't have a feature to send emails to users after profile details were updated. This PR will add this feature

### How to test the changes in this Pull Request:

1. Checkout to this branch and login to my account page.
2. Update the profile details and check whether emails are being received at the user side or not.

### Types of changes:




### Other information:




### Changelog entry

> UR-1049 profile details updated email to user.
